src/units/pathfinder2remaster/player-core/ancestry/dwarf/heritage/heritage-dwarf-ancient-blooded.yml

```diff
@@ -0,0 +1,24 @@
+unit: heritage/dwarf/remaster/ancient-blooded
+in: heritage/dwarf
+group: "_{Player Core}"
+name: "_{Ancient-Blooded Dwarf}"
+
+inc:
+  - set: heritage
+    value: "_{Ancient-Blooded Dwarf}"
+
+  - at: '@heritage'
+    replace:
+      - article:
+        title: "_{Ancient-Blooded Dwarf}"
+        contents:
+          - p: "_{Call on ancient blood to gain a bonus to saving throws.}"
+
+  - at: '@saving-throws'
+    add:
+      - action: reaction
+        contents:
+          - p:
+            title: "_{Call on Ancient Blood}"
+          - value-block: "_{+1}"
+            content: "_{bonus to saving throws against a magical effect, until the end of your turn.}"
```

Comment:
You can shorten this line slightly:
`_{to saves against magical effects this turn}`